CONTRIBUTING.md

```diff
@@ -197,5 +197,22 @@ to merge**.
   - Send staging tag as PR for review.
 - Create a tag `v{version}` on the merged PR and push remote.
 
+## Merging existing ECS conventions
+
+The Elastic Common Schema (ECS) is being merged into OpenTelemetry Semantic
+Conventions per [OTEP 222][otep222]. When adding a semantic convention that
+exists in some form in ECS, consider the following guidelines:
+
+- Prefer using the existing ECS name when possible. In particular:
+  - Do not modify an ECS name for the only reason of complying with
+    OpenTelemetry general attribute guidelines.
```

Comment:
Based on the discussions on `host.ip` we might want to define this a bit more.

Reply:
Do you mean refining as in "we should make it clear what the general attribute guidelines are" or "specifying which attribute guidelines we should follow"?

Reply:
with "general attribute guidelines" I was mostly referring to this: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#general-guidelines. That is the central place we have such guidelines.. and if something is not there, we should add. (for ex #394 is not there yet)

Reply:
The recommendations here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md should also be part of this IMO. If you think it makes sense to explicitly list what guidelines we are talking about I can add an explicit list.

Reply:
@joaopgrassi I think the guidelines @mx-psi linked (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md) are more generic and (should) reference the one you linked.

https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#general-guidelines is only about metrics, right?

Reply:
Right, all of those apply, yes. I think it would be good to link them in here.